docker-compose.yml

```diff
@@ -28,6 +28,7 @@ services:
       - "db:database"
     ports:
       - 9000:9000
+      - 9999:9999
```

Comment:
Thoughts (blocking): this will conflict with our browser testing env, could we choose a different port?

Reply:
Ah shoot, sure thing.

Reply:
It's a popular port